### PR TITLE
feat: mobile feed layout & nav bar

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Changes
- adding a `FeedNav` at the top of the feed layout
- updating footer nav items
- moving reading streaks to the top of the `for you` feed

TODO:
- [x] go through all pages and make sure the layout is not broken
- [x] validate the previous layout still looks the same if the feature flag is disabled

## Previews

Screenshot 2024-03-25 at 10 20 32

---

Screenshot 2024-03-25 at 10 20 20

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

N / A

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-130

### Preview domain
https://mi-130-layout-feed.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/MainLayout.tsx [hunk 1]
```diff
@@ -91,6 +92,7 @@ function MainLayoutComponent({
 
   const isLaptopXL = useViewSize(ViewSize.LaptopXL);
   const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const { useNewMobileLayout } = useMobileUxExperiment();
```
Comment (nensidosari) [inline packages/shared/src/components/MainLayout.tsx:95]: Are we using this here? from changes it looks like we aren't

File: packages/shared/src/components/filters/MyFeedHeading.tsx [hunk 2]
```diff
@@ -103,46 +109,64 @@ function MyFeedHeading({
...
+            }
+          >
+            {!isMobile && !useNewMobileLayout ? 'Feed settings' : null}
+          </Button>
+        </AlertPointer>
+      </>
+    );
+  };
+
+  if (useNewMobileLayout) {
```
Comment (nensidosari) [inline packages/shared/src/components/filters/MyFeedHeading.tsx:155]: We can use conditionalWrapper component for this with no need to extract RenderFeedActions into a component as done now

**Review by nensidosari — COMMENTED** [review]
Looks good, will give it a try on the preview link as well

**Review by sshanzel — COMMENTED** [review]
Looking great overall, I don't have anything major as things have been covered by Nensi's comments.

**Comment by nensidosari** [comment]
Screenshot 2024-03-25 at 4 17 15 PM
Noticed this feed settings button staying in the middle here

**Comment by DragosIorgulescu** [comment]
Closing this as we will be using https://github.com/dailydotdev/apps/pull/2855 to track and review

**Pull request closed without merging**